tests/common/fixtures/__init__.py

```diff
@@ -1,13 +1,15 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from tests.common.fixtures.elasticsearch import es_client
+from tests.common.fixtures.elasticsearch import es_client, es6_client, es_connect
 from tests.common.fixtures.elasticsearch import init_elasticsearch
 from tests.common.fixtures.elasticsearch import delete_all_elasticsearch_documents
 
 
 __all__ = (
     "es_client",
+    "es6_client",
+    "es_connect",
```

Comment:
This doesn't need to be fixed in this PR, but we need to decide what we're doing with the `es_connect` fixture to avoid the confusion of having two different ways to establish a connection and two different connections.

The most obvious solution I can see would be to use the `using=` argument when creating an elasticsearch_dsl `Search` instance and re-use the connection established by the `es_client` fixture.

This will be easier to change though once we are no longer trying to juggle two ES clusters in the tests.